meeko/__init__.py

```diff
@@ -47,21 +55,22 @@
 from .openff_xml_parser import get_openff_epsilon_sigma
 from .hydrate import Hydrate
 
-__all__ = ['MoleculePreparation', 'RDKitMoleculeSetup',
-        'pdbutils', 'geomutils', 'rdkitutils', 'utils',
-        'AtomTyper', 'PDBQTMolecule', 'PDBQTReceptor', 'analysis',
-        'LinkedRDKitChorizo', 'ChorizoResidue', 'ResidueAdditionalConnection',
-        'add_rotamers_to_chorizo_molsetups',
-        'RDKitMolCreate',
-        'PDBQTWriterLegacy',
-        'reactive_typer',
-        'get_reactive_config',
-        'gridbox',
-        'oids_block_from_setup',
-        'parse_offxml',
-        'Hydrate',
-        'Restraint',
-]
+__all__ = ['MoleculePreparation', 'RDKitMoleculeSetup', 'MoleculeSetupEncoder',
+           'pdbutils', 'geomutils', 'rdkitutils', 'utils',
+           'AtomTyper', 'PDBQTMolecule', 'PDBQTReceptor', 'analysis',
+           'LinkedRDKitChorizo', 'ChorizoResidue', 'ResiduePadder', 'ResidueTemplate',
+           'LinkedRDKitChorizoEncoder', 'ChorizoResidueEncoder', 'ResiduePadderEncoder', 'ResidueTemplateEncoder',
```

Comment:
ResidueChemTemplates and the ResidueChemTemplatesEncoder should probably be added here